.circleci/scripts/update_config_template.js

```diff
@@ -28,6 +28,8 @@ const generateE2eMatrix = (templateName, outputName) => {
   console.log('e2ePackagesToTest:', e2ePackagesToTest)
   const e2ePackages = e2ePackagesToTest.filter(pkg => !adaptersWithJava.includes(pkg))
   const e2ePackagesWithJava = e2ePackagesToTest.filter(pkg => adaptersWithJava.includes(pkg))
+  const e2ePackagesWithPrettyName = e2ePackages.map((pkg) => { return `${pkg.split('/').pop()}:${pkg}` })
+  const e2ePackagesWithJavaWithPrettyName = e2ePackagesWithJava.map((pkg) => { return `${pkg.split('/').pop()}:${pkg}` })
```

Comment:
I think this is better, less verbose and leads to a smaller diff.
```suggestion
  const e2ePackages = e2ePackagesToTest.filter(pkg => !adaptersWithJava.includes(pkg)).map(pkg => `${pkg.split('/').pop()}:${pkg}`)
  const e2ePackagesWithJava = e2ePackagesToTest.filter(pkg => adaptersWithJava.includes(pkg)).map(pkg => `${pkg.split('/').pop()}:${pkg}`)
```